x/withdraw/keeper/grpc_query.go

```diff
@@ -0,0 +1,24 @@
+package keeper
+
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/tharsis/evmos/v2/x/withdraw/types"
+)
+
+var _ types.QueryServer = Keeper{}
+
+// Params returns the module parameters
+func (k Keeper) Params(
```

Comment:
kind of sucks that grpc queries are restricted to what we store on chain. Would be nice to have indexable data like an ORM offers that lets us see which addresses did the ibc withdraw process, without having to write whole other indexer service... (would require some kind of opt-in data store that indexes as the chain goes, would be really nice for observability).

Reply:
cc @jolube